csharp/src/Ice/SSLEngine.cs

```diff
@@ -10,9 +10,7 @@
 using System.Security.Cryptography;
 using System.Security.Cryptography.X509Certificates;
 
-using ZeroC.Ice;
-
-namespace ZeroC.IceSSL
+namespace ZeroC.Ice
 {
     internal class SSLEngine
```

Comment:
Should be SslEngine for consistency with the new `Ssl` prefix.

Reply:
Maybe now is a good time to rename ssl to something else.

Reply:
Fixed, we can do the renaming in a follow-up PR